### PR TITLE
sync folder "develop" with perl-dbd-avatica

### PR DESCRIPTION
**Pull request by uid66**
sync folder "develop" with perl-dbd-avatica and remove workdir from main image

### CODE REVIEW

File: develop/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN export PATH=$PATH:$(dirname $(${CCPREFIX}gcc -print-prog-name=cc1))
 RUN ln -s /usr/lib/$(gcc -dumpmachine)/libstdc++.so.6 /usr/lib/$(gcc -dumpmachine)/libstdc++.so
 
 # perl modules
-ADD . /client
```
Comment (Logioniz) [inline develop/Dockerfile]: It seems that without mount folder this container is useless.

**Review by Logioniz — COMMENTED** [review]
Why did you move develop/{app|client}/Dockerfile to develop/Dockerfile?
I followed this logic: each container has its own directory for building.

File: develop/Dockerfile (hunk 1 above)
Comment (uid66) [reply develop/Dockerfile]: No, no need to copy all files of repo to image.
Following Dockerfile commands require only cpanfile and dist.ini. Main app service from docker-compose.yml uses its own volume with repo.

Comment (Logioniz) [reply develop/Dockerfile]: Your words only confirm what I said. Container is useless without volumes.

**Review by Logioniz — COMMENTED** [review]
Looks OK.

**Pull request merged**